internal/controller/pkg/revision/deployment_test.go

```diff
@@ -0,0 +1,161 @@
+package revision
```

Comment:
@chlunde thanks again for adding tests here :) could you make sure to add the appropriate license header to this file?